### DISCUSSION TITLE
React-location docs

### DISCUSSION THREAD

**Discussion by wenchonglee** [TanStack/router#607] [General]
Hi,

Not sure if it is hidden somewhere, but are `react-location` docs still hosted somewhere?
While waiting for the router library to release, I'm still using `react-location` and need to use it for references.

I'm currently using the wayback machine to access it, but I think it should be fair for the old docs to still be visible until `@tanstack/router` is released? If it is up somewhere, maybe it can be more visible in the new docs?

Thanks!

**Comment by rlesniak** [comment]
Bump. @tannerlinsley what happened to them? Many people still using this version

**Comment by rfdrake27** [comment]
I'm going to second this. Removing the docs is a major problem for the following reasons: 

1) Not everyone has the time or capability to migrate over to @tanstack/router on large projects.

2) @tanstack/router is still in beta. I regularly check in on the discord and see comments about @tanstack/router docs being missing or out of date, bugs for path matching, etc. Right now I feel like I'm being pushed to adopt something that feels unfinished just because the rug was pulled out from under react-location.

3) Tanner has a lot on his plate right now. It is understandable that Nozzle takes priority, but as a business I need to be confident that I am not moving over to another router that has unsolved issues with uncertainty on when fixes could be made. 

Simply maintaining the old react-location docs with a big banner that it is legacy/unmaintained/whatever is far better than pulling them off of the internet. Frankly it makes me nervous staying on @tanstack anything if I can't trust that documentation will be around. If react-query gets a refresh am I going to have this same problem?

**Reply by SeanCassiere** [reply]
> [https://github.com/TanStack/router/tree/react-location/docs/src/pages](https://github.com/TanStack/router/tree/react-location/docs/src/pages)

**Comment by tannerlinsley** [comment]
The docs are still on GitHub https://github.com/TanStack/router/tree/main/docs/src/pages

I’m in the process of moving them to the new TanStack site as the previous version of Router. React Query will never be affected by any of my trailblazing in actively developing projects, rest assured.

I just launched my products at Nozzle and we’ll have some more time to finish Router now. Just got my tonsils out, but after I heal up a bit, expect more active development on both Router and love for existing Location users.

Tanner Linsley

**Reply by adamgajzlerowicz** [reply]
Hi, still trying to find that migration guide from @tanstack/react-location

**Reply by SeanCassiere** [reply]
For the time being, all *react-location* [docs](https://github.com/TanStack/router/tree/react-location/docs/src/pages) are under a branch with the same name.

**Comment by nick318** [comment]
@tannerlinsley I want to highlight again the importance of the react-location docs being hosted on the internet, providing a migration path to the new @Tanstack/router. The decision to pull it off and create a redirect for URL https://react-location.tanstack.com/ to the https://tanstack.com/router/latest creates trust issues in the community. I know that docs are under [this branch](https://github.com/TanStack/router/tree/react-location?tab=readme-ov-file), but it is not a solution. To work with them efficiently you have to host it locally.

**Reply by tannerlinsley** [reply]
Part of the issue here is that the hosted version of the docs got a bit mangled beyond my repair. We would likely need to stand up something basic and new. The redirect you're seeing is an edge redirect via cloudflare.

**Comment by SeanCassiere** [comment]
A migration guide for moving from React Location to TanStack Router can be found here: https://tanstack.com/router/latest/docs/framework/react/migrate-from-react-location

This is not a comprehensive guide, but rather a concise overview of the key steps and common changes involved in migrating from React Location to TanStack Router.
